@kiva/kv-components/vue/KvWwwHeader.vue

```diff
@@ -142,14 +147,31 @@ export default {
 			type: String,
 			default: '/ui-login',
 		},
+		balance: {
+			type: Number,
+			default: 0,
+		},
+		isBorrower: {
+			type: Boolean,
+			default: false,
+		},
+		isTrustee: {
+			type: Boolean,
+			default: false,
+		},
 	},
-	setup() {
+	emits: [
+		'load-lend-menu-data',
+	],
+	setup(props, { emit }) {
 		const linksVisible = ref(true);
 		const searchBar = ref(null);
 		const searchOpen = ref(false);
 		const activeHeaderItem = ref(null);
 		const menuOpen = ref(false);
-		const menuComponent = ref(null);
+		const menuComponent = shallowRef(null);
+		const menuComponentInstance = ref(null);
+		const userId = ref(props.userId);
```

Comment:
userId should be declared as a prop, and then it will be available in the template and won't need to be declared here.